cdap-security/src/main/java/co/cask/cdap/security/guice/SecureStoreModules.java

```diff
@@ -66,6 +70,26 @@ protected void configure() {
     };
   }
 
+  private static Object getStore(final CConfiguration cConf, Injector injector) {
+    if (AbstractSecureStore.isKMSBacked(cConf)) {
```

Comment:
reverse this condition and to return FileSecureStore here, so you can reduce indentation